src/main/resources/com/microsoft/azure/vmagent/AzureVMComputer/configure.jelly

```diff
@@ -3,22 +3,23 @@
   <l:layout permission="${it.EXTENDED_READ}" title="${it.name} Configuration">
     <st:include page="sidepanel.jelly"/>
     <l:main-panel>
+      <j:set var="instance" value="${it.node}" />
+      <j:set var="descriptor" value="${instance.descriptor}" />
+      <j:set var="readOnlyMode" value="true" />
       <l:app-bar title="${it.name} Configuration" />
-      <f:entry title="${%Name}">
-        <f:readOnlyTextbox value="${it.name}"/>
-      </f:entry>
+      <p><a href="${it.azurePortalLink}">View on Azure Portal</a></p>
```

Comment:
With the node properties this was pushed too far down, moving to the top keeps it easily discoverable